src/props.fonts.css

```diff
@@ -1,7 +1,19 @@
 :where(html) {
-  --font-sans: system-ui,-apple-system,Segoe UI,Roboto,Ubuntu,Cantarell,Noto Sans,sans-serif;
-  --font-serif: ui-serif,serif;
-  --font-mono: Dank Mono,Operator Mono,Inconsolata,Fira Mono,ui-monospace,SF Mono,Monaco,Droid Sans Mono,Source Code Pro,monospace;
```

Comment:
actually, 1 more comment, as changing these names means i need to make a breaking change release. alternatively, we could keep these same names or just keep the original props, that way it's a minor version bump and no one's apps break.

which is better?
1. keep the previous props in additional to the new ones
1. create vars that map the old names to the most relevant new ones
1. keep the previous props and make the new ones an additional/optional import

i'll toss this into the discord too, see what folks think!

Reply:
I think option 1 or 2 is best. I kinda prefer 2 but I appreciate the hesitation to have breaking changes so I'd not complain at either. In my opinion these are small enough that I don't think we need to hide them behind a flag, and I love the idea of having the prototyping power of these well thought out props easily available. Also most people who care about the css weight should probably be using jit.

Reply:
My vote is to map the old props them to the new ones and remove them in the next major version. Maybe add a deprecation notice to the doc site?